doc/running/running_nest.rst

```diff
@@ -180,6 +180,25 @@ The above code querying for ``receptor_types`` gives a list of port names and NE
 
 For a full example, please see `iaf_psc_exp_multisynapse_vectors.nestml <https://github.com/nest/nestml/blob/master/tests/nest_tests/resources/iaf_psc_exp_multisynapse_vectors.nestml>`_ for the neuron model and ``test_multisynapse_with_vector_input_ports`` in `tests/nest_tests/nest_multisynapse_test.py <https://github.com/nest/nestml/blob/master/tests/nest_tests/nest_multisynapse_test.py>`_ for the corresponding test.
 
+Multiple continuous input ports in NEST
+---------------------------------------
+
+See :ref:`Multiple input ports` to specify multiple continuous input ports in a neuron.
```

Comment:
```suggestion
```

Reply:
I kept this line as it is, as the reference points to the example of how to use it. I changed the name of the currents to match that example.